docs/UserGuide.md

```diff
@@ -12,6 +12,17 @@
 - [Product Information](#product-information)
 - [Quick Start](#quick-start)
 - [Commands](#commands)
+  1) [Registering an Account](#registering-an-account-coderegistercode)
+  2) [Login into Account](#login-into-account-codelogincode)
+  2) [Logout from Account](#logout-from-account-codelogoutcode)
+  2) [Adding a Contact](#adding-a-contact-codecode)
+  2) [Deleting a Contact](#deleting-a-contact-code-code)
+  3) [Editing a Contact](#editing-a-contact-codecode)
+  4) [Toggling the display](#toggling-the-display-codecode)
+  5) [Finding Contacts by Name](#finding-contacts-by-name-codecode)
+  5) [Listing all matching information](#listing-all-matching-information-codelscode)
+  6) [Undoing a Command](#undoing-a-command-codeundocode)
+  7) [Redoing a Command](#redoing-a-command-coderedocode)
```

Comment:
These hyperlinks don't work.

Reply:
i dont think u can test the hyperlink by the code file, need to open in intellij or on the website then can check